src/ElectricPokemon.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private String type = "electric";
+    List<String> attacks = Arrays.asList("thunderPunch", "electroBall", "thunder", "voltTackle");
+
+    public ElectricPokemon(String food, int hp, int level, String name, String sound) {
+        super(food, hp, level, name, sound);
+        this.setType(type);
+    }
+
+    @Override
+    List<String> getAttacks() {
+        return attacks;
+    }
+
+    void thunderPunch(Pokemon name, Pokemon enemy) {
```

Comment:
Waarom heb je deze `package private` en niet `public`?

Reply:
De `Pokemon name` parameter zou je hier ook weg kunnen laten en gewoon `this` gebruiken